src-tauri/src/carpe_error.rs

```diff
@@ -24,30 +26,62 @@ impl From<anyhow::Error> for CarpeError {
     }
 }
 
-const E_MISC: u8 = 100;
+impl From<TxError> for CarpeError {
+    fn from(e: TxError) -> Self {
+      // check if the is a tower error
+      match tower::tower_errors::parse_error(e){
+        tower::tower_errors::TowerError::Unknown => CarpeError::tx("unknown tx error submitting tower"),
+        tower::tower_errors::TowerError::Other(v) => CarpeError::tx(&format!("unknown tx error submitting tower, message: {}", &v)),
```

Comment:
Wondering if unknown and Other could combined into one, It's confusing somehow.

Reply:
yes this would be a change in Libra repo though.